README.md

```diff
@@ -42,3 +42,42 @@ use OmniAuth::Builder do
     }
  )
 ```
+
+## Bypassing OmniAuth/OAuth
+
+It is also possible to bypass omniauth/ entirely, which can be useful in circumstances where hostnames are dynamic, e.g. in review deployments.  To do this add the following code to your OmniAuth initializer.
+
+```ruby
+if ENV.has_key? 'BYPASS_AUTH' # This environment variable can be changed to whatever you like.
```

Comment:
Just as a note - we've been using BYPASS_OAUTH in a couple of places - not sure if we need the env name to be more specific?

Reply:
It can be whatever you like.  The variable isn't used in this gem at all.

Reply:
Also it isn't specific to OAuth -- **all** auth processed by omniauth is bypassed.